_pages/rfc-unify-coc-enforcement.md

```diff
@@ -0,0 +1,70 @@
+# RFC - Unify CoC and Enforcement policies
+
+## tl;dr
+
+* Move the enforcement steps to the top of the CoC.
+* Keep the separate enforcement document as a detailed policy that includes lessons/values/nuance.
+* Stop directing people to both the CoC and enforcement policy: on the signup page, on docs, etc.
+* Put a link to the full enforcement values doc in the CoC.
+
+## Intro
+
+The first thing people should see when they go to the CoC is how it affects them. The bulk of the stuff on the CoC itself amounts to legalese that most users won't read -- so in the interest of conveying information that matters right away, it's best to lead with how CoC violations are handled. I would argue that's the main thing people even are about.
+
+Hopefully, these changes will make expectations even clearer for everone!
+
+## Changes
+
+### Move enforcement steps to CoC
+
+There's two sections of enforcement steps in the enforcement doc. Combine them, reword them, and put them on the very top of the main CoC.
+
+```
+If you see a CoC violation, these are the steps that should be followed:
+
+1. Tell the person “hey, that’s not ok”.
```

Comment:
I know it's not easy for everyone but I do like that this encourages someone noticing a CoC violation be pro-active about commenting on it. Do you think this write-up should give examples? (For example, something along the lines of: "If you don't feel comfortable addressing a person who made a problematic remark in the channel, feel free to PM them.) I think it empowers the person noticing a remark to manage the situation and the conversation might not need to be escalated to admin.